packages/cli-tools/src/cacheManager.ts

```diff
@@ -4,7 +4,12 @@ import os from 'os';
 import appDirs from 'appdirsjs';
 import logger from './logger';
 
-type CacheKey = 'eTag' | 'lastChecked' | 'latestVersion' | 'dependencies';
+type CacheKey =
+  | 'eTag'
+  | 'lastChecked'
+  | 'latestVersion'
+  | 'dependencies'
+  | 'lastUsedDeviceId';
```

Comment:
I think it would be good to add the same feature for Android in the future - with that in mind, can we name the cache key accordingly to the plaftorm?

Reply:
✅